user/advanced-configuration/config-files.md

```diff
@@ -122,8 +125,19 @@ Currently supported settings:
 
 -   `secure_copy_sequence` and `secure_paste_sequence` - key sequences used to trigger secure copy and paste.
 
--   `windows_count_limit` - limit on concurrent windows.
 
--   `audio_low_latency` - force low-latency audio mode (about 40ms compared to 200-500ms by default).
-    Note that this will cause much higher CPU usage in dom0.
```

Comment:
This parameter still exists. Although it's enabled by default, so description could be updated (disabling it may save some CPU in dom0).